src/test/java/pages/PlayListPage.java

```diff
@@ -0,0 +1,261 @@
+    LoginPage loginPage = new LoginPage();
+
+    // Locators create New PlayList
+    //navigate PlayList
+    By createPlayListBtn = By.cssSelector("[title='Create a new playlist'] ");
+    // click playList
+    By clickPlayList = By.cssSelector("[title='Create a new playlist']");
+      // navigate New playList
+      By navigateNewPlayL = By.cssSelector("[data-testid='playlist-context-menu-create-simple'] ");
+      // click New PlayL
+       By clickNewPlayL = By.cssSelector("[data-testid='playlist-context-menu-create-simple'] ");
+       // navigate field create New playList
+       By navigateFildNewPlayL = By.xpath("//form[@name='create-simple-playlist-form']//input");
+       // create New name Playlist
+       By createNewName = By.xpath("//form[@name='create-simple-playlist-form']//input ");
+       // Assert New name playList
+       By assertNewName = By.cssSelector(".success.show");
+
+       // Locators Delete PlayList
+      By navigateNewPL = By.xpath("//li[@class='playlist playlist']");
+      // right click PlayList
+       By rightClicPL = By.xpath("//li[@class='playlist playlist']");
+       // navigate delete Btn
+      By navigateDeletePLBtn = By.xpath(".//*[@class='menu playlist-item-menu']//li[contains (text(), 'Delete')]");
+      // click delete Btn
+      By clickDeleteBtn = By.xpath(".//*[@class='menu playlist-item-menu']//li[contains (text(), 'Delete')]");
+      // Assert playList deleted
+      By assertPLDeleted = By.cssSelector(".success.show");
+
+
+
+
+    // Created PlayList
+    public void navigatePlayListBtn() {
+        WebElement createPlayListButton = driver.findElement(createPlayListBtn);
+        new Actions(driver)
+                .contextClick(createPlayListButton)
+                .perform();
+    }
+
+
+    public void clickPlayListBtn() {
+        WebElement clickPlayListButton = driver.findElement(clickPlayList);
+        new Actions(driver)
+                .click(clickPlayListButton)
+                .perform();
+    }
+
+    public void navigateNewPlaylistBtn() {
+        WebElement newPlayListButton = driver.findElement(navigateNewPlayL);
+        new Actions(driver)
+                .contextClick(newPlayListButton)
+                .perform();
+    }
+    public void clickNewPlayListBtn() {
+        WebElement clickNewPlayListButton = driver.findElement(clickNewPlayL);
+        new Actions(driver)
+                .click(clickNewPlayListButton)
+                .perform();
+    }
+    public void navigateField() {
+        WebElement navigateFieldCreatePlayList = driver.findElement(navigateFildNewPlayL);
+        new Actions(driver)
+                .contextClick(navigateFieldCreatePlayList)
+                .perform();
+    }
+    public void newNamePlayList() throws InterruptedException {
+        WebElement addplayListNameField = driver.findElement(createNewName);
+        addplayListNameField.click();
+        addplayListNameField.clear();
+        Thread.sleep(1000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.


1) 
![image](https://github.com/Test-Pro-Bootcamp/A42-QA-Automation-Framework/assets/104761482/93fc34bf-756e-403d-92ce-624761543df6)


2)
 
![image](https://github.com/Test-Pro-Bootcamp/A42-QA-Automation-Framework/assets/104761482/20d1dc2e-ecfd-4d12-8937-3146df18e5aa)